### PR TITLE
About iterations

### PR DESCRIPTION
**Pull request by heatherlarsen**
### :pushpin: Summary

If merged, this PR updates some of the content for the About Overview, Support, and Contribution pages.

### :hammer_and_wrench: Detailed description
The review included checking for the following:
- Proper grammar
- Consistent and simple language/terms
- Proper casing on headings
- More clear headings
- Content flow
- Brevity

### :link: External links

Jira ticket: [HDS-1349](https://hashicorp.atlassian.net/browse/HDS-1349)

***

### 👀 Reviewer's checklist:

- [ ] +1 Percy if applicable
- [ ] Confirm that PR has a changelog update via [Changesets](https://github.com/changesets/changesets) if needed

:speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.

[HDS-1349]: https://hashicorp.atlassian.net/browse/HDS-1349?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: website/docs/about/support.md [hunk 1]
```diff
@@ -1,25 +1,27 @@
...
 ---
 
+!!! Info
 
-We are happy to provide support to HashiCorp product teams that use our design system. Here are some of the ways to get help if you get stuck.
+Many resources and links on this page are only available to HashiCorp employees.
+!!!
 
-## Office Hours
+We're happy to provide support to HashiCorp product teams that use Helios. Use these resources to get help if you get stuck.
```
Comment (Dhaulagiri) [inline website/docs/about/support.md:11]:
```suggestion
We’re happy to provide support to HashiCorp product teams that use Helios. Use these resources to get help.
```
Including @alex-ju 's suggestion but also removing the "if you get stuck" as you don't need to be "stuck" to seek out help from us.

File: website/docs/about/overview.md [hunk 2]
```diff
@@ -7,43 +7,45 @@ order: 101
...
 
-## Our consumers
+## Who is Helios for
 
-HashiCorp develops and maintains a number of different products with varying sub-brands and approaches to the user experience. In an effort to bring these different product lines and teams towards a common visual language and single source of truth, the Helios Design System was born.
+To bring the different product lines and teams towards a common design language, Helios is available and recommended for HashiCorp product teams. We work closely with these teams on integration and implementation while collecting feedback to understand consumer needs better and inform our [roadmap](https://go.hashi.co/hds-rollout).
 
-Helios has been adopted at varying levels by all products within the R&D organization. We work closely with these teams at the integration and implementation level while collecting feedback to better understand consumer needs and inform our [roadmap](https://go.hashi.co/hds-rollout).
+Helios does not currently support HashiCorp marketing teams.
```
Comment (Dhaulagiri) [inline website/docs/about/overview.md:40]: isn't developer a "marketing" team?

**Review by jorytindall — APPROVED** [review]

File: website/docs/about/overview.md (hunk 2 above)
Comment (heatherlarsen) [reply website/docs/about/overview.md:40]: I chatted with Brayden about this a bit, and they're in a weird spot with the team being split across design/eng. I'll wordsmith this a bit more.

File: website/docs/about/support.md [hunk 3]
```diff
@@ -1,25 +1,27 @@
...
+!!!
 
-## Office Hours
+We’re happy to provide support to HashiCorp product teams that use Helios. Use these resources to get help if you get stuck.
 
-We offer office hours as a way to get feedback on your designs, or help you get unstuck in a code migration.
+## Live support
 
-[Add your agenda item](https://docs.google.com/forms/d/e/1FAIpQLSf9T_M7gSAQ7zTM9z0gR11kvqpiTnbByrTsqDk305s3SxblRw/viewform) to the next office hours session (internal employees only).
+Join the [#team-design-systems](https://hashicorp.slack.com/archives/C7KTUHNUS) Slack channel to ask questions, request and provide feedback, or start a conversation with our team. As a globally-distributed team, there may be a slight delay in answering some questions, but we're committed to responding in a timely manner.
```
Comment (KristinLBradley) [inline website/docs/about/support.md:15]:
```suggestion
Join the [#team-design-systems](https://hashicorp.slack.com/archives/C7KTUHNUS) Slack channel to ask questions, request and provide feedback, or start a conversation with our team. As a globally-distributed team, there may be a slight delay in answering some questions, but we’re committed to responding in a timely manner.
```

**Review by KristinLBradley — APPROVED** [review]

**Comment by heatherlarsen** [comment]
@Dhaulagiri I updated the Overview section about consumers a bit. Let me know what you think.

**Comment by Dhaulagiri** [comment]
@heatherlarsen LGTM, thanks!

**Pull request merged**